devicetypes/smartthings/ecobee-thermostat.src/ecobee-thermostat.groovy

```diff
@@ -153,13 +167,12 @@ def parse(String description) {
 }
 
 def refresh() {
-	log.debug "refresh"
-	poll()
+	log.debug "refresh, calling parent poll"
+	parent.poll()
 }
 
 void poll() {
-	log.debug "Executing 'poll' using parent SmartApp"
-	parent.pollChild()
+	log.debug "poll not implemented as it is done by parent SmartApp every 5 minutes"
```

Comment:
should we remove this since it is not being called anywhere?

Reply:
unfortunately this method may be called by SA and routines as once upon a time the polling capability was defined for this DTH. removing this command will result in missing method exceptions if there are existing SA and routines calling it.